tests/apps/tests.py

```diff
@@ -78,7 +78,8 @@ def test_no_such_app(self):
         with self.assertRaises(ImportError):
             with self.settings(INSTALLED_APPS=['there is no such app']):
                 pass
-        with self.assertRaises(ImportError):
+        msg = "Cannot import 'there is no such app', check AppConfig.name is correctly defined."
```

Comment:
@timgraham 
I could change it to something like:

`"Cannot import '%s' check '%s.%s.name' is correctly defined." % (app_name, mod_path, cls_name)`

which would render to:

`"Cannot import 'there is no such app' check 'apps.apps.NoSuchApp.name' is correctly defined."`

Look good?